src/Microsoft.Health.Dicom.Core/Features/Retrieve/RetrieveResourceService.cs

```diff
@@ -123,9 +127,10 @@ public async Task<RetrieveResourceResponse> GetInstanceResourceAsync(RetrieveRes
             // transcoding of single instance
             if (needsTranscoding)
             {
-                _logger.LogInformation("Transcoding Instance");
                 FileProperties fileProperties = await RetrieveHelpers.CheckFileSize(_blobDataStore, _retrieveConfiguration.MaxDicomFileSize, version, instance.VersionedInstanceIdentifier.Partition.Name, false, cancellationToken);
                 SetTranscodingBillingProperties(fileProperties.ContentLength);
+                _logger.LogInformation("Transcoding Instance {Path} of size {ContentLength}", fileProperties.ContentLength, fileProperties.Path);
```

Comment:
Do we want to log path? Today it is just watermark but once IDP is enabled it can contain any information